GPU/Directx9/VertexShaderGeneratorDX9.cpp

```diff
@@ -232,7 +232,7 @@ void GenerateVertexShaderDX9(int prim, char *buffer, bool useHWTransform) {
 			WRITE(p, "%s", boneWeightAttrDecl[TranslateNumBones(vertTypeGetNumBoneWeights(vertType))]);
 		}
 		if (doTexture && hasTexcoord) {
-			if (doTextureProjection)
+			if (doTextureProjection && !throughmode)
```

Comment:
Hmm.  I'm a little confused.  GLES only does vec3 for sw transform, is it even correct to ever have a float3 here?

-[Unknown]